ext/pdo/pdo_stmt.c

```diff
@@ -1129,28 +1148,28 @@ static int pdo_stmt_verify_mode(pdo_stmt_t *stmt, zend_long mode, int fetch_all)
 	switch(mode) {
 		case PDO_FETCH_FUNC:
 			if (!fetch_all) {
-				pdo_raise_impl_error(stmt->dbh, stmt, "HY000", "PDO::FETCH_FUNC is only allowed in PDOStatement::fetchAll()");
+				zend_value_error("Can only use PDO::FETCH_FUNC in PDOStatement::fetchAll()");
 				return 0;
 			}
 			return 1;
 
 		case PDO_FETCH_LAZY:
 			if (fetch_all) {
-				pdo_raise_impl_error(stmt->dbh, stmt, "HY000", "PDO::FETCH_LAZY can't be used with PDOStatement::fetchAll()");
+				zend_value_error("Cannot be PDO::FETCH_LAZY in PDOStatement::fetchAll()");
```

Comment:
The "cannot be" formulation would be for `zend_argument_value_error(mod_arg_num)`. Without it, it should be a `"Fetch mode cannot be"` or so.